Translation/en/RedirectedResources/assets/abdata/h/list/01/personality_voice_c35/translation.txt

```diff
@@ -2251,10 +2251,10 @@
 あぁん……おまんこ、まだイッとる感じするぅ……精液の熱さが残っとるけんよ……=Aahn... I feel like my pussy is still climaxing... it's because the heat from your semen is still in there...
 こんな気持ちいいこと……私、完全にハマッたん……すぐに赤ちゃん出来ちゃいそうなん……=Something so pleasurable... I've completely gotten addicted to it... I feel like I'll get pregnant in no time...
 もう……赤ちゃん出来ちゃってもいいよね……だってこげん気持ちいいっちゃもん……=At this point... it'll be fine even if we have a baby, won't it... Afterall, it feels so good to do this...
-えへへ……なんか「やりました！」みたいな感じになるよね……危険日の中出しって♪=Ehehe... It kinda gives you that sort of "We did it!" feeling, doesn't it... doing it inside on a risky day, that is♪
+えへへ……なんか「やりました！」みたいな感じになるよね……危険日の中出しって♪=Ehehe... It kinda gives you that sort of We did it!" feeling, doesn't it... doing it inside on a risky day, that is♪"
 あは……危険日って一番感じる日なんかも……だって……めっちゃ気持ち良かったっちゃもん=Aha... I might be the most sensitive on risky days... I mean... it felt so good, afterall.
 君と一緒にイッて妊娠するんやったら……私、全然平気よ。えへへ……=I'm personally perfectly fine with it, you know... if I get pregnant from cumming together with you. Ehehe...
-危険な日に中出しすると、最後には「妊娠したいっ！」って思うっちゃん。女子やけんかな……=When you're about to cum inside me on a risky day, I can't help but shout "I wanna get pregnant!" inside my head near the end. I wonder if it's because I'm a girl...
+危険な日に中出しすると、最後には「妊娠したいっ！」って思うっちゃん。女子やけんかな……=When you're about to cum inside me on a risky day, I can't help but shout I wanna get pregnant!" inside my head near the end. I wonder if it's because I'm a girl..."
```

Comment:
```suggestion
危険な日に中出しすると、最後には「妊娠したいっ！」って思うっちゃん。女子やけんかな……=When you're about to cum inside me on a risky day, I can't help but shout "I wanna get pregnant!" inside my head near the end. I wonder if it's because I'm a girl...
```